React/Base/RCTUtils.m

```diff
@@ -740,11 +740,14 @@ BOOL RCTIsLocalAssetURL(NSURL *__nullable imageURL)
 
   if (!image) {
     // Attempt to load from the file system
-    NSString *filePath = [NSString stringWithUTF8String:[imageURL fileSystemRepresentation]];
-    if (filePath.pathExtension.length == 0) {
-      filePath = [filePath stringByAppendingPathExtension:@"png"];
+    const char* fileSystemCString = [imageURL fileSystemRepresentation];
+    if (fileSystemCString != nil) {
+      NSString *filePath = [NSString stringWithUTF8String:[imageURL fileSystemRepresentation]];
```

Comment:
Why not use `fileSystemCString` here?

Reply:
My bad! I forgot to review the code there.